conf/salt/project/django/init.sls

```diff
@@ -21,3 +21,20 @@ manage:
       - pip: pip_requirements
       - file: project_path
       - file: log_dir
+
+run_with_db:
+  file.managed:
+    - name: {{ vars.path_from_root('run_with_db.sh') }}
+    - source: salt://project/django/run_with_db.sh
+    - user: {{ pillar['project_name'] }}
+    - group: {{ pillar['project_name'] }}
+    - mode: 700
+    - template: jinja
+    - context:
+        settings: "{{ pillar['project_name']}}.settings.{{ pillar['environment'] }}"
+        virtualenv_root: "{{ vars.venv_dir }}"
+        directory: "{{ vars.source_dir }}"
```

Comment:
We don't appear to be using any of this context in the template. I think we can just drop it.